sdk/ml/azure-ai-ml/azure/ai/ml/entities/_builders/sweep.py

```diff
@@ -149,6 +150,30 @@ def trial(self):
         """Id or instance of the command component/job to be run for the step."""
         return self._component
 
+    @property
+    def search_space(self):
```

Comment:
docstring, copy what's in original component or job object : 

e.g.
Dictionary of the hyperparameter search space. The key is the name of the
        hyperparameter and the value is the parameter expression.